editor/package.json

```diff
@@ -27,11 +27,11 @@
-    "production": "WEBPACK_MODE='production' NODE_OPTIONS=--max_old_space_size=6656 webpack --define process.env.REACT_APP_ENVIRONMENT_CONFIG=\"'production'\" --define process.env.REACT_APP_AUTH0_CLIENT_ID=\"'$AUTH0_CLIENT_ID'\" --define process.env.REACT_APP_AUTH0_ENDPOINT=\"'$AUTH0_ENDPOINT'\" --define process.env.REACT_APP_AUTH0_REDIRECT_URI=\"'$AUTH0_REDIRECT_URI'\" --define process.env.REACT_APP_COMMIT_HASH=\"'$UTOPIA_SHA'\" --define process.env.GOOGLE_WEB_FONTS_KEY=\"'$GOOGLE_WEB_FONTS_KEY'\"",
-    "staging": "WEBPACK_MODE='staging' NODE_OPTIONS=--max_old_space_size=6656 webpack --define process.env.REACT_APP_ENVIRONMENT_CONFIG=\"'staging'\" --define process.env.REACT_APP_AUTH0_CLIENT_ID=\"'$AUTH0_CLIENT_ID'\" --define process.env.REACT_APP_AUTH0_ENDPOINT=\"'$AUTH0_ENDPOINT'\" --define process.env.REACT_APP_AUTH0_REDIRECT_URI=\"'$AUTH0_REDIRECT_URI'\" --define process.env.REACT_APP_COMMIT_HASH=\"'$UTOPIA_SHA'\" --define process.env.GOOGLE_WEB_FONTS_KEY=\"'$GOOGLE_WEB_FONTS_KEY'\"",
+    "move-fast-and-break-things": "REACT_APP_ENVIRONMENT_CONFIG=development REACT_APP_AUTH0_CLIENT_ID=$AUTH0_CLIENT_ID REACT_APP_AUTH0_ENDPOINT=$AUTH0_ENDPOINT REACT_APP_AUTH0_REDIRECT_URI=$AUTH0_REDIRECT_URI REACT_APP_COMMIT_HASH=`git rev-parse HEAD` NODE_OPTIONS=--max_old_space_size=8192 webpack-dev-server --progress",
+    "performance-test": "NODE_ENV=production REACT_APP_ENVIRONMENT_CONFIG=development REACT_APP_AUTH0_ENDPOINT=$AUTH0_ENDPOINT REACT_APP_AUTH0_REDIRECT_URI=$AUTH0_REDIRECT_URI REACT_APP_COMMIT_HASH=`git rev-parse HEAD` NODE_OPTIONS=--max_old_space_size=8192 PERFORMANCE='true' webpack-dev-server --progress",
+    "move-fast-and-break-things-hot": "REACT_APP_ENVIRONMENT_CONFIG=development REACT_APP_AUTH0_CLIENT_ID=$AUTH0_CLIENT_ID REACT_APP_AUTH0_ENDPOINT=$AUTH0_ENDPOINT REACT_APP_AUTH0_REDIRECT_URI=$AUTH0_REDIRECT_URI REACT_APP_COMMIT_HASH=`git rev-parse HEAD` NODE_OPTIONS=--max_old_space_size=8192 HOT='true' webpack-dev-server --progress",
+    "production": "REACT_APP_ENVIRONMENT_CONFIG=production REACT_APP_AUTH0_CLIENT_ID=$AUTH0_CLIENT_ID REACT_APP_AUTH0_ENDPOINT=$AUTH0_ENDPOINT REACT_APP_AUTH0_REDIRECT_URI=$AUTH0_REDIRECT_URI REACT_APP_COMMIT_HASH=$UTOPIA_SHA WEBPACK_MODE='production' REACT_APP_COMMIT_HASH=`git rev-parse HEAD` NODE_OPTIONS=--max_old_space_size=6656 webpack",
+    "staging": "REACT_APP_ENVIRONMENT_CONFIG=staging REACT_APP_AUTH0_CLIENT_ID=$AUTH0_CLIENT_ID REACT_APP_AUTH0_ENDPOINT=$AUTH0_ENDPOINT REACT_APP_AUTH0_REDIRECT_URI=$AUTH0_REDIRECT_URI REACT_APP_COMMIT_HASH=$UTOPIA_SHA WEBPACK_MODE='staging' REACT_APP_COMMIT_HASH=`git rev-parse HEAD` NODE_OPTIONS=--max_old_space_size=6656 webpack",
```

Comment:
The GOOGLE_WEB_FONTS_KEY env is missing from these, is it not used anymore?

Reply:
good catch. it is used, however, that one did not need renaming.

Reply:
I don't understand this comment. So was accidentally removed and needs to be put back before this can be merged?

Reply:
so the env vars listed here are the ones that need renaming / aliasing, because the "outside" env var is called, for example AUTH0_ENDPOINT but the "internal" env var is named REACT_APP_AUTH0_ENDPOINT

Reply:
Riiight ok I'm with you now :+1: